src/api/spec/controllers/webui/users/bs_requests_controller_spec.rb

```diff
@@ -1,26 +1,148 @@
 require 'rails_helper'
 
 RSpec.describe Webui::Users::BsRequestsController do
-  let(:user) { create(:confirmed_user, login: "tom") }
+  let!(:user) { create(:confirmed_user, login: "tom") }
```

Comment:
Simply because it easier to just add the exclamation mark for all the `lets` that need to be eagerly evaluated.

For example in both of these specs the `another_user` let block is eagerly evaluated:
Spec1:
```ruby
let!(:user) { create(:confirmed_user, login: "tom") }
let(:another_user) { create(:confirmed_user, login: "moi") }
let!(:request3) { create(:bs_request, creator: another_user, commenter: another_user) }
```

Spec2:
```ruby
let!(:user) { create(:confirmed_user, login: "tom") }
let!(:another_user) { create(:confirmed_user, login: "moi") }
let!(:request3) { create(:bs_request, creator: another_user, commenter: another_user) }
```

However the Spec1 requires slightly more work and I didn't see the advantage in it either. I dont mind not using `!` if we dont have to but its just my preference to always use `!` if we know the let needs to be eager evaluated.